frontend/src/components/Shared/Navbar.jsx

```
@@ -1,118 +1,117 @@
-					) : (
-						<FontAwesomeIcon icon={faBars} />
-					)}
-				</div>
-			</div>
+    return (
+        <div className="navbar bg-[#323d5e] sticky top-0 z-50">
+            {/* Menu Button */}
+            <div
+                className="absolute left-4 lg:hidden text-white"
+                onClick={toggleMenu}
+            >
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)